h2o-docs/src/product/automl.rst

```diff
@@ -68,9 +68,14 @@ Optional Miscellaneous Parameters
 
 - `max_runtime_secs_per_model <data-science/algo-params/max_runtime_secs_per_model.html>`__: Specify the max amount of time dedicated to the training of each individual model in the AutoML run. Defaults to 0 (disabled). Note that setting this parameter can affect AutoML reproducibility.
 
+- **modeling_plan**: The list of modeling steps to be used by the AutoML engine (they may not all get executed, depending on other constraints).
+
+- `monotone_constraints <data-science/algo-params/monotone_constraints.html>`__: A mapping that represents monotonic constraints. Use +1 to enforce an increasing constraint and -1 to specify a decreasing constraint. Note that constraints can only be defined for numerical columns. This option can only be used when the distribution is either gaussian or bernoulli.
```

Comment:
Remove "Note that constraints can only be defined for numerical columns. This option can only be used when the distribution is either gaussian or bernoulli."